src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -1,12 +1,18 @@
 package mate.academy.hibernate.relations;
 
 import java.util.List;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
```

Comment:
Ensure that the `MovieDaoImpl` class properly handles exceptions, as mentioned in the previous review.